### DISCUSSION TITLE
doc/info/%E8%A7%86%E9%A2%91%E8%BD%AF%E4%BB%B6%E4%B8%8B%E8%BD%BD

### DISCUSSION THREAD

**Discussion by giscus** [zuihou/zuihou-admin-cloud#294] [企业版]
# doc/info/%E8%A7%86%E9%A2%91%E8%BD%AF%E4%BB%B6%E4%B8%8B%E8%BD%BD

微服务中后台快速开发平台

https://tangyh.top/doc/info/%E8%A7%86%E9%A2%91%E8%BD%AF%E4%BB%B6%E4%B8%8B%E8%BD%BD.html

**Comment by TangGuoNiuBi** [comment]
微信群失效啦，麻烦再分享下
